src/App.js

```diff
@@ -1,26 +1,131 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { Component } from 'react';
 import './App.css';
+import 'bulma/css/bulma.css';
+import foods from './foods.json';
+import AddFood from './components/AddFood';
+import Foodbox from './components/FoodBox';
+import Menu from './components/Menu';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+
+class App extends Component {
+  state = {
+    foodList: foods,
+    filteredList: foods,
+    showForm: false,
+    todaysFood:[],
+    allCalories:[]
+  };
+
+  handleAdd = (event) => {
+    event.preventDefault();
+    console.log(event.target);
+
+    let food = event.target;
+    let newFood = {
+      name: food.name.value,
+      calories: food.calories.value,
+      quantity: 0,
+      image: food.image.value,
+    };
+
+    this.setState({
+      foodList: [...this.state.foodList, newFood],
+      filteredList: [...this.state.filteredList, newFood],
```

Comment:
by putting the element first you can see the element being added to the beginning of the list instead of the end.